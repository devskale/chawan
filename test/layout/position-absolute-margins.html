<!DOCTYPE html>
<style>
* {
margin: 0; padding: 0;
</style>
test
<div style="position: absolute; top: 0; left: 0; margin-left: 1ch; margin-top: 1em">
with top/left
</div>
<div style="position: absolute; margin-left: 1ch; margin-top: 1em">
without top/left
</div>
