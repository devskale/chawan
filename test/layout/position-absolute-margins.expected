test
 with top/left
 without top/left
